content/docs/setup/kubernetes/getting-started/index.md

```diff
@@ -1,41 +1,42 @@
 ---
 title: Getting Started
-description: Download and install Istio.
+description: Download, install, and try out Istio.
 weight: 5
 skip_toc: true
-keywords: [kubernetes]
+keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-If you’re new to Istio and just want to try it out, the quickest way to get started is
-by downloading and installing Istio's built-in **demo** configuration.
-Istio can also be installed and customized in many other ways, depending on your platform and intended use.
-Check out our [general installation instructions](/docs/setup/) for details.
-
-To get started, proceed as follows:
+This guide lets you try out Istio quickly and it's the ideal starting
+point if you are new to the project. To get started, download and install
+Istio's built-in **demo** configuration:
 
 1. [Setup your platform](/docs/setup/kubernetes/platform-setup/)
 1. [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release)
 1. [Follow the quick-start installation instructions](/docs/setup/kubernetes/install/kubernetes)
 
-Once you've installed Istio, a good way to start evaluating Istio's features is using our
-[Bookinfo sample application](/docs/examples/bookinfo/).
-You can use this sample to experiment with Istio’s features for traffic routing,
-fault injection, rate limiting, etc..
+With Istio installed, you can [deploy the Bookinfo sample](/docs/examples/bookinfo/#if-you-are-running-on-kubernetes)
+and evaluate Istio's features for traffic routing, fault injection, rate
+limiting, etc. Explore the various [Istio tasks](/docs/tasks/), and try out the
+tasks that interest you. If you are not sure where to begin, the following
+tasks are a good place to start:
+
+- [Request routing](/docs/tasks/traffic-management/request-routing/)
+- [Fault injection](/docs/tasks/traffic-management/fault-injection/)
+- [Traffic shifting](/docs/tasks/traffic-management/traffic-shifting/)
+- [Querying metrics](/docs/tasks/telemetry/metrics/querying-metrics/)
+- [Visualizing metrics](/docs/tasks/telemetry/metrics/using-istio-dashboard/)
+- [Collecting logs](/docs/tasks/telemetry/logs/collecting-logs/)
+- [Rate limiting](/docs/tasks/policy-enforcement/rate-limiting/)
+- [Ingress gateways](/docs/tasks/traffic-management/ingress/ingress-control/)
+- [Accessing external services](/docs/tasks/traffic-management/egress/egress-control/)
+- [Visualizing your mesh](/docs/tasks/telemetry/kiali/)
+
+These tasks provide a good basic understanding of Istio's features. Now, you
```

Comment:
This is missing a very important part of the original text. After these tasks, the user can
```
proceed to explore Istio's many other tasks or, even better, start to deploy your own applications.
```
Thinking about reinstalling and customizing Istio would only come after that.

Reply:
I moved it after the tasks. Readers usually want to jump straight in and, after the tasks, they will want to customize for their use cases.

Reply:
Yes, but you are missing the key point: `explore Istio's many other tasks`. The tasks listed here are just a good starting subset. They should definitely consider exploring more tasks (depending on their interest) and/or experiment with there own apps, **before** they start thinking about customizing the Istio installation.

Reply:
Understood, I'll change the language to make that clear.